packages/react-bindings/src/FocusZone/FocusZone.types.ts

```diff
@@ -38,9 +38,7 @@ export interface IFocusZone {
 /**
  * FocusZone component props.
  */
-export interface FocusZoneProps
-  extends FocusZoneProperties,
-    React.HTMLAttributes<HTMLElement | FocusZone> {
```

Comment:
`React.HTMLAttributes<HTMLElement | FocusZone>` is really strange and looks broken.

It also creates collisions with other typings as `HTMLAttributes` includes callbacks like `onClick` and if `FocusZone` is included it creates `onClick: (e: FocusZone)` 💣